themes/apiarybasestyles/source/css/buttons.styl

```diff
@@ -10,3 +10,10 @@
     background-color #264653
     &:hover
       background-color #2A9D8F
+
+.loadingIndicator
+  $loadingIndicator()
+  &--big
+    $loadingIndicator(size: 12px)
+  &--blue
+    $loadingIndicator(color: $primaryColor)
```

Comment:
yet another example of using `@extends` here...

instead of directly creating a large pile of CSS, it could be a mixin using `@extends` inside itself.